satpy/etc/areas.yaml

```diff
@@ -1385,3 +1422,93 @@ robinson:
   area_extent:
     lower_left_xy: [-2049911.5256036147, 5326895.725982913]
     upper_right_xy: [2049911.5256036168, 8625155.12857459]
+
+# ----------------------------------------------------------------------------------------------------------------------
+# ------------------------------------------------ Areas to be deprecated ----------------------------------------------
+# ----------------------------------------------------------------------------------------------------------------------
+# This section contains areas that are obsolete.
```

Comment:
Is it possible for satpy to issue a warning when someone uses an obsolete area name?

Reply:
Not currently, but I think we could add it. The cases where someone does `new_scn = scn.resample('obsolete_area')` would be very easy to handle. The cases where someone is manually specifying a path to the areas.yaml file and requesting it might be harder since there are a few utilities doing that and most are in pyresample which doesn't know or care about the areas defined in satpy.

Reply:
I guess yes, e.g. a check for deprecated areas could be placed in `satpy.resample.get_area_def()`, which is what is used to get the destination AreaDefinition out of the yaml when resampling... TBD in #819